tests/test_99_regressions.py

```diff
@@ -428,3 +428,36 @@ def dst(self, dt):
     roundtrip = PGPSignature.from_blob(str(pgpsig))
 
     assert pgpsig.created.utctimetuple() == roundtrip.created.utctimetuple()
+
+
+@pytest.mark.regression(issue=341)
+def test_spurious_dash_escapes():
+    from pgpy import PGPKey, PGPMessage
+
+    message_data = '''-----BEGIN PGP SIGNED MESSAGE-----
```

Comment:
Can you make this a raw string (`r'''`) please

Reply:
Sure, done and force pushed.